assets/admin/listItemActions/AlertNameItemAction.js

```diff
@@ -0,0 +1,18 @@
+import {translate} from 'sulu-admin-bundle/utils';
+import {AbstractListItemAction} from 'sulu-admin-bundle/views';
+
+export default class AlertNameItemAction extends AbstractListItemAction {
+    getItemActionConfig(item) {
+        const {disabled_ids: disabledIds = []} = this.options;
+
+        return {
+            icon: 'su-bell',
```

Comment:
Available icons are documented in the [Sulu Javascript Docs](https://jsdocs.sulu.io/latest/#icon).